components/widgets/land-cover/tree-cover-density/index.js

```diff
@@ -0,0 +1,67 @@
+import { getTreeCoverDensity } from 'services/analysis-cached';
+
+import {
+  POLITICAL_BOUNDARIES_DATASET,
+  TROPICAL_TREE_COVER,
+} from 'data/datasets';
+import {
+  DISPUTED_POLITICAL_BOUNDARIES,
+  POLITICAL_BOUNDARIES,
+  TROPICAL_TREE_COVER_METERS,
+  TROPICAL_TREE_COVER_HECTARE,
+} from 'data/layers';
+
+import getWidgetProps from './selectors';
+
+export default {
+  widget: 'treeCoverDensity',
+  title: 'Tree cover density in {location}',
+  categories: ['land-cover'],
+  types: ['country', 'wdpa', 'aoi'],
+  admins: ['adm0', 'adm1', 'adm2'],
+  large: true,
+  visible: ['dashboard', 'analysis'],
+  chartType: 'composedChart',
+  colors: 'density',
+  settingsConfig: [
+    {
+      key: 'landCategory',
+      label: 'Land Category',
+      type: 'select',
+      placeholder: 'All categories',
+      clearable: true,
+      border: true,
+    },
+  ],
+  pendingKeys: ['threshold', 'years'],
```

Comment:
I think you could remove the elements and leave an empty array here, as these options don't exist.